docs/data-ingestion/Proxmox.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Proxmox
+weight: 1
+sort: 1
+menu:
+  docs:
+    identifier: "proxmox"
+    parent: "data-ingestion"
+    weight: 1
+    # sort: 1
+aliases:
+  - /data-ingestion/proxmox.html
+  - /data-ingestion/Proxmox.html
+---
+
+# Proxmox Data Ingestion
+Since Proxmox Virtual Environment(PVE) and Proxmox Backup Server(PBS) support sending data using the InfluxDB We can use the InfluxDB write support built into VictoriaMetrics
```

Comment:
```suggestion
Since Proxmox Virtual Environment(PVE) and Proxmox Backup Server(PBS) support sending data using the InfluxDB We can use the InfluxDB write support built into VictoriaMetrics.
```
Would it make sense to add links to `Proxmox Virtual Environment` and `Proxmox Backup Server`?